Podfile

```diff
@@ -12,7 +12,7 @@ def wordpress_authenticator_pods
   pod 'Gridicons', '~> 1.0'
   pod 'WordPressUI', '~> 1.7.0'
   pod 'WordPressKit', '~> 4.0-beta.0' # Don't change this until we hit 5.0 in WPKit
-  pod 'WordPressShared', '~> 1.0-beta.0' # Don't change this until we hit 2.0 in WPShared
+  pod 'WordPressShared', '~> 1.9-beta' # Don't change this until we hit 2.0 in WPShared
```

Comment:
># Don't change this until we hit 2.0 in WPShared

What is that about?

Reply:
I believe it was to relax the pod inter-dependency restrictions. However, the Shared method is in version `1.9.1`, so I bumped Shared to the lowest possible version so I could use the `serifFontForTextStyle` method.